workbench-core/example/infrastructure/scripts/cleanup.sh

```diff
@@ -32,13 +35,17 @@ Usage: `basename $0`
     -d [DELETE_DYNAMO_DB_TABLE]
     -r [AWS REGION]
     -p [DELETE_SSM_PARAMETERS]
+    -i [GET_USER_POOL_ID]
+    -t [GET_DYNAMO_DB_TABLE]
     -c [CLEANUP FLAG]
 
 Example:
-    Delete UserPool:        `basename $0` -u 'us-east-1_abcde' -r 'us-east-1' -c
-    Delete DynamoDB Table:  `basename $0` -d 'TABLE_NAME' -r 'us-east-1' -c
-    Delete Ssm Parameter:   `basename $0` -r 'us-east-1' -p -c
-    Delete All:             `basename $0` -u 'us-east-1_abcde' -d 'TABLE_NAME' -r 'us-east-1' -p -c"
+    Get UserPoolId:                     `basename $0` -r 'us-east-1' -i -c
+    Get DynamoDBTable:                  `basename $0` -r 'us-east-1' -t -c
```

Comment:
Ah I see now. Im not sure if the cleanup script is the right place to have `get X variable` options